src/api/spec/factories/project.rb

```diff
@@ -171,5 +171,9 @@
         end
       end
     end
+
+    factory :staging_project do
+      sequence(:name, [*'A'..'Z'].cycle) { |n| n }
```

Comment:
[Nitpick] I would change this accordingly to the creation of the StagingWorflow model.
Right now we create 2 staging projects that are subprojects of the actual one, something like *:Staging:A, *Staging:B

So, it could be something like:
```suggestion
      sequence(:name, [*'A'..'Z'].cycle) { |letter| "#{project.name}:Staging:#{letter}" }
```

Reply:
> [Nitpick] I would change this accordingly to the creation of the StagingWorflow model. [...]

`sequence(:name` modified. I had to add the parameter `:workflow_project_name`. Please have a look.